### DISCUSSION TITLE
Alternative for STATUSMAP

### DISCUSSION THREAD

**Discussion by kyob** [sni/thruk#1217] [General]
Any working alternative for STATUSMAP after upgrade to THRUK 3.0 ?

**Comment by sni** [comment]
Which one are you missing? Because i found them all pretty useless, at least with starting with a medium sized number of hosts/services.

**Reply by kyob** [reply]
Over 500 hosts with dependency parent/child. I need to visualize connections between them. I was useing STATUSMAP but after upgrade to Thruk 3.0 is stops working.

**Reply by white-line** [reply]
> Over 500 hosts with dependency parent/child. I need to visualize connections between them. I was useing STATUSMAP but after upgrade to Thruk 3.0 is stops working.

Try this - [nagios4_features](https://github.com/white-line/thruk_nagios4_features)

**Comment by sni** [comment]
I added some basic dependency viewer based on visjs yesterday. Nothing too fancy but a
quick overview on dependencies, ex.:
https://demo.thruk.org/demo/thruk/cgi-bin/extinfo.cgi?type=dtree&host=naemon1&service=ping&backend=40863

**Reply by kyob** [reply]
Looks fine for me.
